docroot/modules/custom/mass_translations/mass_translations.module

```diff
@@ -23,9 +23,9 @@ use Drupal\node\NodeInterface;
  * Implements hook_preprocess_media().
  */
 function mass_translations_preprocess_media(&$variables) {
+  $cache_tags = [];
   $media = $variables['media'];
   if (($variables["view_mode"] == 'media_download_link' || $variables["view_mode"] == 'linked_desc' || $variables["view_mode"] == 'manual_desc') && $media->hasField('field_media_english_version')) {
-    $cache_tags[] = 'media-list:document';
```

Comment:
What do you offer instead? I don't see any code that helps us clear the caches of the multilanguage documents widget.